frontend/packages/dev-console/src/utils/resource-label-utils.ts

```diff
@@ -14,11 +14,13 @@ export const getAppLabels = (
   };
 };
 
-export const getAppAnnotations = (gitURL: string, gitRef: string) => {
-  const ref = gitRef || 'master';
+export const getAppAnnotations = (gitURL: string, gitRef: string, cheURL: string) => {
   return {
     'app.openshift.io/vcs-uri': gitURL,
-    'app.openshift.io/vcs-ref': ref,
+    'app.openshift.io/vcs-ref': gitRef || 'master',
+    ...(cheURL && {
+      'app.openshift.io/edit-url': `${cheURL}/f?url=${gitURL}&policy.create=peruser`,
```

Comment:
can we change ```policy.create```  to ```policys.create``` ? To match https://github.com/eclipse/che/issues/14355 ?

Reply:
Sure, will update the code.

Reply:
Done.

Reply:
thx